SeleniumPracticalExercise/TestCases/AddEmployee.cs

```diff
@@ -1,31 +1,54 @@
 ﻿using NUnit.Framework;
+using SeleniumPracticalExercise.Common;
+using SeleniumPracticalExercise.PageObjects;
 using SeleniumPracticalExercise.TestCases.Common;
 
-namespace SeleniumPracticalExercise.TestCases
-{
-    class AddEmployee : BaseTestLocal
+class AddEmployee : BaseTestLocal
     {
         [Test]
         [Category("Add Employee")]
         public void AddEmployeeTest()
         {
+            string username = "Admin"; 
+            string password = "admin123"; 
+            string firstName = Utils.GenerateRandomString(6); 
+            string lastName = Utils.GenerateRandomString(8);
+
             // Steps to automate:
```

Comment:
Don't forget to remove all of these commented steps for the test case as we discussed before.

Reply:
painful for me to do but I did it